packages/components/dropdown/src/DropdownItems.tsx

```diff
@@ -5,23 +5,26 @@ import { useDropdownContext } from './DropdownContext'
 
 interface ItemsProps {
   children: ReactNode
+  className?: string
 }
 
-export const Items = ({ children }: ItemsProps) => {
+export const Items = ({ children, className, ...props }: ItemsProps) => {
   const { isOpen, getMenuProps, hasPopover, setLastInteractionType } = useDropdownContext()
 
   return (
     <ul
+      className={cx(
+        className,
+        'flex flex-col',
+        isOpen ? 'block' : 'pointer-events-none opacity-0',
+        hasPopover && 'p-lg'
+      )}
+      {...props}
```

Comment:
This is to enable this component to be used `asChild` (to merge its props with its parent props)